browser/src/control/Toolbar.js

```diff
@@ -584,6 +584,21 @@ L.Map.include({
 				this.filterResults(searchTerm, isAnyMatchingContent, id);
 			}
 		}.bind(this));
+
+
+		const buttons = document.querySelectorAll('.scroll-button');
```

Comment:
Just fyi: We can use something like:
document.getElementById('online-help-content').querySelectorAll(':scope .scroll-button');

So it would pick only the elements inside the 'online-help-content' elements with class '.scroll-button'.

Didn't test this but it should work.